satellite/satellitedb/migrate.go

```diff
@@ -341,6 +341,28 @@ func (db *DB) PostgresMigration() *migrate.Migration {
 					)`,
 				},
 			},
+			{
+				Description: "Change bucket_id type",
+				Version:     6,
```

Comment:
Version must always be incremented with each step... but you can do multiple SQL statements in a `migrate.SQL`, it's a list of strings.